k2/csrc/fsa_utils.cu

```diff
@@ -120,14 +119,23 @@ static void SplitStringToVector(const std::string &in, const char *delim,
    the final state and set its label to -1.
 
    @param [in]  is    The input stream that contains the acceptor.
-
+   @param  [in]  num_aux_labels  The number of auxiliary labels to expect
+                     per arc; 0 == acceptor, 1 == transducer, but may be more.
+   @param [out] aux_labels_out  If num_aux_labels > 0, this will be
+                     assigned to,  with a new array on CPU of shape
+                     (num_arcs, num_aux_labels).
```

Comment:
The comment says its shape is `(num_arcs, num_aux_labels)`, but the implementation
at line 193 is
```cpp
    *aux_labels_out = Array2<int32_t>(c, num_aux_labels,
                                      (int32_t)arcs.size());
```
The code stores the `aux_labels` per arc non-contiguously. Is it intentional?
Just feel it is unnatural.

Reply:
Comment was wrong.  It is intentional, because the array will need to be separated into multiple arrays for different aux_label names.